pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -241,8 +241,12 @@ func (rq *ResourceQuotaController) addQuota(obj interface{}) {
 // worker runs a worker thread that just dequeues items, processes them, and marks them done.
 func (rq *ResourceQuotaController) worker(queue workqueue.RateLimitingInterface) func() {
 	workFunc := func() bool {
+		if queue.Len() == 0 {
```

Comment:
> If the queue is empty, they will try again 1s later.

this might help dealing w/ duplicated items, while it would also cost a 1s delay at most even if the items are unique on their arrival, amiright?

Reply:
Right. Of course, in our clusters, we care more about the success rate of creation, quota updates are acceptable once 1s.